contrib/protobuf/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "protobuf"
```

Comment:
this isn't really split correctly, https://github.com/chimera-linux/cports/pull/165 does it completely instead